notification_lib/novu_manager/subscribers_manager.py

```diff
@@ -178,6 +185,66 @@ def update_subscriber_credentials(
 
         cls.handle_response(response, "Can't set subscriber credentials !").json()
 
+    @classmethod
+    def get_subscriber_preferences(cls, subscriber_id: str) -> list[subscriberTemplatePreferencesType]:
+        """Get subscriber preferences for all templates."""
+
+        response = cls.send_request(
+            operation="GET",
+            endpoint=f"/v1/subscribers/{subscriber_id}/preferences",
+        )
+
+        json_response = \
```

Comment:
how this has passed pre-commit ?

Reply:
is it about the \ ?

Reply:
yes

Reply:
it's to mark that what's underneath is on the same line, even blake does sometimes format your code to this format